nanoc/spec/nanoc/regressions/gh_1378_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+describe 'GH-1378', site: true, stdio: true do
```

Comment:
Totally cool to learn how to assemble a "real site" instead of having to mock all Nanoc internals 👍

Reply:
This is in particular used for regression tests, although I have also started writing integration tests in `spec/nanoc/integration`.

It’s not a substitute for unit tests, though, because the integration/regression tests tend to run slowly and so it’s not great for testing all the different cases.